provision-osx/tasks/install_ruby.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+-
+  name: Copy prerequisite gpg key
+  get_url:
+    url: "{{rvm_gpg_url}}"
+    dest: "{{remote_tmp_path}}/{{rvm_gpg_file_name}}"
+  environment:
+    http_proxy: "{{ proxy_url | default('') }}"
+    https_proxy: "{{ proxy_url | default('') }}"
+
+-
+  name: Install rvm prerequisites
+  shell: "/usr/local/bin/gpg --import {{remote_tmp_path}}/{{rvm_gpg_file_name}}"
+  args:
+    executable: /bin/bash
+  register: list
+
+-
+  name: Retrieve rvm install script
+  get_url: 
+    url: "{{rvm_install_url}}"
+    dest: "{{remote_tmp_path}}/{{rvm_install_file_name}}"
+  register: rvm_prerequisites
+  environment:
+    http_proxy: "{{ proxy_url | default('') }}"
+    https_proxy: "{{ proxy_url | default('') }}"
+
+-
+  name: Install rvm
+  shell: "bash {{remote_tmp_path}}/{{rvm_install_file_name}} stable"
```

Comment:
Just to note if any of these scripts call out to the internet and use external tools we may need to set the environment on them as well so its worth checking